addons/compact-buttons/addon.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "Compact Buttons",
```

We don't usually use Start Case in addon names as of now.
```suggestion
    "name": "Compact buttons",
```